protractor-docker/docker-selenium-grid-conf.js

```diff
@@ -0,0 +1,19 @@
+exports.config = {
+	framework: 'jasmine2'
+	, specs: ['specs/*.js']
+ 	, seleniumAddress: 'http://localhost:4444/wd/hub'
+	,  capabilities: {
+		browserName: 'chrome'
+		, shardTestFiles: true
+		, maxInstances: 5
```

Comment:
I think we should remove shardTestFiles and maxInstances. These are not necessarily docker specific. However, in you feel like it is, you should add a comment about why these values are set.